src/main/java/seedu/address/ui/AppointmentCard.java

```diff
@@ -56,5 +66,31 @@ public AppointmentCard(AppointmentView apptView, int displayedIndex) {
         timePeriod.setText(appt.getTimePeriod().getStartTime() + " - " + appt.getTimePeriod().getEndTime());
         appointmentType.setText(appt.getAppointmentType().toString());
         note.setText("Notes: " + appt.getNote().toString());
+        Timeline timeline = getTimeline();
+        timeline.play();
+    }
+
+    private Timeline getTimeline() {
+        EventHandler<ActionEvent> appointmentMissedEventHandler = event -> {
+            bindCardPaneStyle();
+        };
+        KeyFrame keyframe = new KeyFrame(Duration.seconds(1), appointmentMissedEventHandler);
+        Timeline timeline = new Timeline(keyframe);
+        timeline.setCycleCount(Animation.INDEFINITE); // Run indefinitely
+        return timeline;
+    }
+
+    private void bindCardPaneStyle() {
+        cardPane.styleProperty().bind(
+                Bindings.when(isBeforeCurrentTime(appt))
+                        .then("-fx-background-color: #FF7074")
+                        .otherwise("-fx-background-color: defaultColor")
+        );
+    }
+
+    private BooleanBinding isBeforeCurrentTime(Appointment appt) {
+        return Bindings.createBooleanBinding(() ->
+            appt.getTimePeriod().getStartTime().value.isBefore(LocalTime.now())
```

Comment:
Should we create a getStartTime() method in Appointment? 
I realise there is quite a handful of occasions where we need StartTime from Appointment. Doing so will prevent talking to the friend of a friend.

Reply:
Sounds good! Have made the changes :)